apps/alarm/app.js

```diff
@@ -57,4 +57,9 @@ module.exports = function (activity) {
     AlarmCore.clearReminderTts()
     logger.log(this.appId + ' destroyed')
   })
+
+  // unhandled rejection
+  process.on('unhandledRejection', err => {
+    logger.error('Alarm: Unhandled Rejection', err)
+  })
```

Comment:
This is not a good habit :(

Reply:
It is just a temporary solution, I will delete unhandled rejection when support atomic process.